.github/workflows/release-notes.yml

```diff
@@ -6,34 +6,18 @@ on:
   push:
     tags:
       - v*
-  workflow_run:
-    workflows: ['Tag on Merge from Release Branch']
-    types:
-      - completed
+  workflow_dispatch:
 
```

Comment:
The `workflow_run` is this flaky workflow chaining that we are getting rid of.

I note that a number of releases don't have release notes, so I have added `workflow_dispatch` so that we can add notes to existing releases.